service/src/main/java/greencity/service/EmailServiceImpl.java

```diff
@@ -125,6 +129,16 @@ public void sendNewNewsForSubscriber(List<NewsSubscriberResponseDto> subscribers
 
     @Override
     public void sendCreatedNewsForAuthor(EcoNewsForSendEmailDto newDto) {
+        String email = newDto.getAuthor().getEmail();
+
+        if (email == null || !EMAIL_PATTERN.matcher(email).matches()) {
+            throw new BadRequestException("Wrong format of email");
```

Comment:
It would be better to add @Email annotation in Dto class on the field which contains email.
It had better use check in one place(in dto) instead of writing the same if-else in all methods where will be used this dto with email.